setenv-pro.sh

```diff
@@ -0,0 +1,16 @@
+# source it before launching platform
+# NOTE: this is an addition to .env file, not a standalone file !
+export COMPOSE_PROJECT_NAME=po_pro
+# we do not want postgres on the pro side, we use the one from off
+export COMPOSE_PROFILES=
+# Tweak config for producer platform
+export PRODUCERS_PLATFORM=1
+export MINION_QUEUE=pro.openfoodfacts.localhost
+# use a different port for minion admin
+# Note: we do not change PRODUCT_OPENER_DOMAIN for Config2.pm will handle this
+# neither do we change PRODUCT_OPENER_FLAVOR, as off-pro use same contents as off
+
+set_ps1 () {
+    export PS1=${PS1:-}"(pro) "
+}
+set_ps1;
```

Comment:
maybe instead of this script I would propose a ready-to-go .env.pro file that people can switch to by simply setting `ENV_FILE=.env.pro`: we could then have an `.env.obf`, `.env.opf` etc that has the default setup for each flavor and the setup would not be too complicated.

Reply:
the problem is that you would have to maintain two .env that are very close, and you can't combine .env.

Working on the pro platform is not a thing a lot of contributor will do. So I think it's ok.